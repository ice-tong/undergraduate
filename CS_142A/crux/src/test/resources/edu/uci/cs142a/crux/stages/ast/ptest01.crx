// test look up variable

func main() : void {
  ::printInt(a);
}
