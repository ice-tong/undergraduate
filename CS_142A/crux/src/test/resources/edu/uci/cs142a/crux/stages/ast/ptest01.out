ResolveSymbolError(4,13)[Could not find a.]
